### ISSUE TITLE
Increase Cache more than 1 day

### ISSUE THREAD

**Issue by jacarrasco** [SergeyPirogov/webdriver_manager#132]
Hello, 

Is there any way to extend the default cache for weeks, months or forever? 
I understand that it can be achieved modifying just this section below but the variable should need to be passed in some way. Is it something that can be integrated in this excellent module? What do you think it would be the best to pass the value?

```
def __is_valid(self, driver_info, cache_days=1):
        dates_diff = get_date_diff(driver_info['timestamp'],
                                   datetime.date.today(),
                                   self._date_format)
        return dates_diff < cache_days
```

Thank you

**Comment by SergeyPirogov** [comment]
Yes! sure. It would be great  to have a pull request for this

**Comment by SergeyPirogov** [comment]
Added via https://github.com/SergeyPirogov/webdriver_manager/commit/e7b74fea1be6cd99948918cd1cf56e725d57f3ab

**Comment by jacarrasco** [comment]
Hi,
Thank you for the pull. I tried with below code and it does not work.

`driver = webdriver.Firefox(executable_path=GeckoDriverManager(cache_valid_range=99).install())`

Would it be possible that it is also needed to include the variable in the init of the GeckoDriver?

```
class GeckoDriverManager(DriverManager):
    def __init__(self, version="latest",
                 os_type=utils.os_type(),
                 path=None,
                 name="geckodriver",
                 url="https://github.com/mozilla/geckodriver/releases/download",
                 latest_release_url="https://api.github.com/repos/mozilla/geckodriver/releases/latest",
                 mozila_release_tag="https://api.github.com/repos/mozilla/geckodriver/releases/tags/{0}",
                 log_level=logging.INFO
                 cache_valid_range=1):
```


Best Regards,

**Comment by SergeyPirogov** [comment]
Yes, need to fix this

**Comment by SergeyPirogov** [comment]
Fixed in 3.2.1
